README.md

```diff
@@ -182,8 +182,6 @@ A lot of the earlier details such as what order which locations were unlocked or
 6. Madrid [[v1.12](https://www.ipa4fun.com/ipa/124229/)]
 7. Budapest [[v1.14.1](https://www.ipa4fun.com/ipa/267007/)]
 8. Florence [[v19.0](https://www.ipa4fun.com/ipa/578209/)]
-9. Porto [[v20.1](https://www.ipa4fun.com/ipa/625102/)]
```

Comment:
Wait sorry why are we removing this?

Reply:
I think it'll be better to add it with the rest of the Porto additions

Reply:
Ah good point.
I don't think it's really necessary, especially since we'll add it in later, there's no issues with doing incremental changes :)

Reply:
Wouldn't it be better to have all those incremental changes added in a single batch though? So they'd be grouped together?

Reply:
Ideally yes. But in reality, making that batch change might take a while (and might never get done), or we might miss something in the first batch and need to add it in later on (and thus wouldn't be a single batch anyway).
And since it's already there and correct now, there's no harm in leaving it there.

Reply:
Also to be clear, I understand it's a change you've already made, and I'm sorry that I'm essentially asking you to put more effort in undoing it again.
I'm just trying to avoid you, or someone else going through the effort of adding it back, when the good work's already been done :sweat_smile:

Reply:
Well, I'm already planning on adding it in #3, I feel it'd be ideal to put all the Porto data there for tracking progress as well

Reply:
Ah I see, thanks!